### DISCUSSION TITLE
Azure AI Prompt Flow

### DISCUSSION THREAD

**Discussion by michaeljperry** [microsoft/teams-sdk#1785] [Q&A]
Hey everyone,

I'm new to working with Teams and AI in general.

I am using Azure Open AI to create a proof of concept conversational bot and was able to get a working example by using a model and an Azure Search service to use my own data. That all worked great.

However, I am not able to authenticate once I moved to using prompt flow in Azure AI Studio. I have the endpoint url and the key but when I try to test the bot I get the following error:

The bot encountered an unhandled error: The chat completion API returned an error status of Forbidden: Service request failed. Status: 403 (Forbidden)

Content: key_auth_bad_header_forbidden Please check this guide to understand why this error code might have been returned https://docs.microsoft.com/en-us/azure/machine-learning/how-to-troubleshoot-online-endpoints#http-status-codes

I'm making the assumption that this is because when the request is made to the Prompt Flow endpoint the Authorization header is not populated. I'm still reading through documentation and trying to digest everything, but I also feel that features of prompt flow and features that are available through the Teams AI SDK are overlapping. Would I be better off just using the AI Model and Azure Search instead of Prompt Flow?  Any help or direction on how I might be able to get this to work would be appreciated.

**Comment by lifan0127** [comment]
Regarding Teams AI and Prompt Flow integration, I suppose it is possible to create a Prompt Flow based planner, similar to the [Autogen Planner](https://github.com/microsoft/teams-ai/pull/1787)? So that we can leverage Prompt Flow's LLM tracing capability (related discussion https://github.com/microsoft/teams-ai/discussions/1573)? @corinagum

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
